sdk/core/azure-core/test/ut/operation.cpp

```diff
@@ -88,3 +88,22 @@ TEST(Operation, Status)
   EXPECT_THROW(operation.Value(), std::runtime_error);
   EXPECT_EQ(operation.Status(), OperationStatus::Cancelled);
 }
+
+TEST(Operation, ResumeToken)
+{
+  StringClient client;
+  std::string token;
+  {
+    auto operation = client.StartStringUpdate();
+    token = operation.GetResumeToken();
+  }
+  {
+    for (auto resumedOperation = StringOperation::CreateFromResumeToken(client, token);
+         !resumedOperation.IsDone();
+         resumedOperation.Poll())
+    {
+      EXPECT_FALSE(resumedOperation.HasValue());
+      EXPECT_THROW(resumedOperation.Value(), std::runtime_error);
+    }
```

Comment:
Add asserts on resumed operation after the for loop is done, to make sure `HasValue` is true.

Reply:
That's already part of a previous test.  A resumed operation is no different than the original in terms of implementation.